src/tests/suite/daos_mgmt.c

```diff
@@ -256,13 +256,13 @@ list_pools_test(void **state)
 	/***** Test: retrieve number of pools in system *****/
 	npools = npools_orig = 0xABC0; /* Junk value (e.g., uninitialized) */
 	/* test only */
-	rc = dmg_pool_list(dmg_config_file, arg->group, &npools, NULL);
```

Comment:
I don't feel too strongly about it, but would it make sense to keep at least one dmg_pool_list() invocation in this test (maybe for that instance, calling both dmg and libdaos API versions and verifying both)?

I see that dmg is exercised in catastrophic recovery tests, so there is still usage/coverage there.

Reply:
The `dmg_pool_list()` helper was added when we removed the old client-side pool list capability. The ftest stuff exercises dmg pretty thoroughly, so my inclination would actually be to completely remove the dmg helper in favor of going back to the client-only listing code in the suite.